### PR TITLE
Use test container capability in test projects.

### PR DESCRIPTION
**Pull request by codito**
Update build scripts to not download runtimes if they are already installed.

### CODE REVIEW

File: scripts/build/TestPlatform.Settings.targets [hunk 1]
```diff
@@ -52,6 +52,11 @@
         <GenerateDocumentationFile>false</GenerateDocumentationFile>
       </PropertyGroup>
 
+      <!-- Test projects are not discovered in test window without test container capability -->
+      <ItemGroup>
```
Comment (Faizan2304) [inline scripts/build/TestPlatform.Settings.targets]: This file will get included in product code as well. Shouldn't we add this only for test project?

**Review by Faizan2304 — APPROVED** [review]

File: scripts/build/TestPlatform.Settings.targets [hunk 2]
```diff
@@ -52,6 +52,11 @@
         <GenerateDocumentationFile>false</GenerateDocumentationFile>
       </PropertyGroup>
 
+      <!-- Test projects are not discovered in test window without test container capability -->
```
Comment (smadala) [inline scripts/build/TestPlatform.Settings.targets]: Please remove `<Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />`. I think tests should get discovered.

File: scripts/build/TestPlatform.Settings.targets (hunk 1 above)
Comment (codito) [inline scripts/build/TestPlatform.Settings.targets]: It is under a TestProject condition, see Line 50.

**Review by smadala — APPROVED** [review]

File: scripts/build/TestPlatform.Settings.targets (hunk 1 above)
Comment (Faizan2304) [inline scripts/build/TestPlatform.Settings.targets]: Ohh yes.. My bad 👍

**Pull request merged**